node.go

```diff
@@ -181,17 +185,37 @@ func (n *Node) Index(index *SearchIndex) error {
 			text = append(text, stringified)
 			// TODO: Index other the parts of the node:
 			// - assets: read exif data?
+		// We probably need to guard against non-meta.yaml yaml here,
+		// but this is just a first go.
+		case ".yaml":
```

Comment:
You probably want to add the meta date outside the doc iteration using the getter methods, i.e. n.Description(), n.Tags() and friends.